sites/docs/src/content/docs/tutorials/adding_a_pipeline/first_release.md

```diff
@@ -36,12 +36,16 @@ Use `nf-core pipelines lint --release`. This will check that your code conforms
 Ok - now the tough bit - does your workflow stand up to the scrutiny of the nf-core team?!
 Not to worry, we're a friendly bunch.
 
-To get your pipeline reviewed for its initial release, do the following:
+To start a review for the initial release of your pipeline:
 
-1. Make a pull-request from the `dev` branch to `master` on the nf-core fork. This is a
+1. Create a new branch named `TEMPLATE_<TOOLS_VERSION>`, based on the version of `TEMPLATE` from the `dev `branch you plan to merge into `master`.
```

Comment:
I never create this branch, I just make a the PR directly to the `TEMPLATE`, why would I need the versioned branch, @maxulysse ?